### PR TITLE
EUI-5539 WA-filter logic implimented for SSCS

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5539

### Change description ###

Change to what services can be selected in the Show work filter in My work

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/work-allocation-2/components/task-list-filter/task-list-filter.component.spec.ts [hunk 1]
```diff
@@ -8,13 +8,15 @@ import { By } from '@angular/platform-browser';
...
-import { StoreModule } from '@ngrx/store';
+import { StoreModule, Store } from '@ngrx/store';
+import { provideMockStore } from '@ngrx/store/testing';
 import { of } from 'rxjs/internal/observable/of';
 
 import { LocationDataService, WASupportedJurisdictionsService, WorkAllocationTaskService } from '../../services';
 import { TaskTypesService } from '../../services/task-types.service';
 import { ALL_LOCATIONS } from '../constants/locations';
 import { TaskListFilterComponent } from './task-list-filter.component';
+import * as fromAppStore from '../../../app/store';
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/components/task-list-filter/task-list-filter.component.spec.ts:19]: Import should be at top of other '../' imports

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor changes requested

File: src/app/app.constants.ts [hunk 2]
```diff
@@ -136,4 +136,4 @@ export const LD_FLAG_REMOVE_USER_FROM_CASE_MC: string = 'remove-user-from-case-m
 export const LD_FLAG_MC_APPLICATION_THEMES: string = 'mc-application-themes';
 export const LEGAL_OPS_ROLE_LIST: string[] = ['caseworker-ia-caseofficer', 'caseworker-ia', 'caseworker-ia-admofficer', 'task-supervisor'];
 export const JUDICIAL_ROLE_LIST: string[] = ['caseworker-ia-iacjudge'];
-export const SERVICE_OPTIONS_LIST = [{key: 'IA', label: 'Immigration and Asylum'}];
+export const SERVICE_OPTIONS_LIST = [{key: 'IA', label: 'Immigration and Asylum'}, {key: 'SSCS', label: 'Social security and child support'}];
```
Comment (udaydenduluri33) [inline src/app/app.constants.ts:139]: Unuderstand this is only added for testing purposes

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by connorpgpmcelroy — DISMISSED** [review]
Comments have been worked on so all good

**Review by udaydenduluri33 — DISMISSED** [review]
@tarikdogan-hmcts  Thanks for making the changes

**Pull request closed without merging**